ethcore/res/ethereum/tests-issues/currents.json

```diff
@@ -0,0 +1,215 @@
+{ "block":
+        [
+
+                {
+                        "reference": 9568,
```

Comment:
Can you change this type to a string? Sometimes it may be easier to reference it via `ethereum/tests`'s issue number.